README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
+- Encourage good application behavior across platforms.
+- Automate platform resolution independence.
+- Enable robust internationalization.
+- Ensure accessibility to a wide group of people.
+- Produce small and fast binaries with low memory usage.
+- Have a small dependency tree with a high quality code base and good organization.
+- Focus on powerful, desktop-grade applications.
```

Comment:
this is a longer-term goal, and I'm a bit hesitant about including it here; this might be a good discussion though, of like what are a good set of intermediate goals?

For instance, right now I think a good goal would be "make it easy to create a simple UI for your rust application".

Reply:
I think this was included to contrast with mobile.

Reply:
Yep, it is about support for complex interactions which are needed by more powerful applications (as which I would classify Runebender for example).